gossip/evmstore/store.go

```diff
@@ -103,14 +104,17 @@ func (s *Store) initCache() {
 	s.cache.EvmBlocks = s.makeCache(s.cfg.Cache.EvmBlocksSize, s.cfg.Cache.EvmBlocksNum)
 }
 
-func (s *Store) CreateEvmSnapshot(root common.Hash) (err error) {
+func (s *Store) GenerateEvmSnapshot(root common.Hash, rebuild, async bool) (err error) {
```

Comment:
just a coding convention, it's clearer if we specific datatype for each parameter separately. 
By the way, I'm OK with the current one if everyone feel good with it :) 
 
```suggestion
func (s *Store) GenerateEvmSnapshot(root common.Hash, rebuild bool, async bool) (err error) {
```